### PR TITLE
Initial entity management mappings

### PR DESCRIPTION
**Pull request by i509VCB**
There are a few unnamed classes but I feel those need a further look for their purpose or discussion on proper names.

### CODE REVIEW

File: mappings/net/minecraft/class_5569.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5569
```
Comment (OroArmor) [inline mappings/net/minecraft/class_5569.mapping:1]: Maybe `TrackableEntity`?

File: mappings/net/minecraft/class_5570.mapping [hunk 2]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5570
```
Comment (OroArmor) [inline mappings/net/minecraft/class_5570.mapping:1]: `TrackedClientEntity`

File: mappings/net/minecraft/class_5579.mapping [hunk 3]
```diff
@@ -0,0 +1,32 @@
+CLASS net/minecraft/class_5579
```
Comment (OroArmor) [inline mappings/net/minecraft/class_5579.mapping:1]: `TrackedEntityWriter`

File: mappings/net/minecraft/class_5584.mapping [hunk 4]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_5584
```
Comment (OroArmor) [inline mappings/net/minecraft/class_5584.mapping:1]: TrackableEntityStatus

**Review by OroArmor — COMMENTED** [review]
These are all some ideas from me quickly checking in on the code.

File: mappings/net/minecraft/class_5570.mapping (hunk 2 above)
Comment (i509VCB) [reply mappings/net/minecraft/class_5570.mapping:1]: What? This is not client only at all

Comment (OroArmor) [reply mappings/net/minecraft/class_5570.mapping:1]: I might have reviewed the wrong class

**Review by YanisBft — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Pull request merged**